test/prototype_common_utils.py

```diff
@@ -53,45 +54,31 @@ def __init__(
         actual,
         expected,
         *,
-        agg_method=None,
-        allowed_percentage_diff=None,
+        mae=False,
```

Comment:
I ported both `agg_method` and `allowed_percentage` diff from the stable tests. In the prototype tests we only ever used `agg_method="mean"`. This just makes it more expressive by adding an `mae=False` flag instead.